napari-workshops/notebooks/manual_annotation.md

```diff
@@ -45,34 +44,33 @@ from napari.utils import nbscreenshot
 viewer = napari.Viewer()
 ```
 
-In this notebook we will load our data directly into napari using out builtin plugin reader
+In this notebook we will use the same `cells3d` data set:
 
 ```{code-cell} ipython3
-:tags: [remove-output]
-
-viewer.open('data/nuclei.tif')
-```
+from skimage.data import cells3d
 
-This loads the 3D data into the napari viewer. 
-Scrolling to the 30th z-slice should look as follows:
+image_data = cells3d()  # shape (60, 2, 256, 256)
 
-```{code-cell} ipython3
-:tags: [remove-cell]
+membranes = image_data[:, 0, :, :]
+nuclei = image_data[:, 1, :, :]
 
-viewer.dims.current_step = (30, 0, 0)
+viewer.add_image(nuclei)
 ```
 
+This loads the 3D data into the napari viewer on the 30th slice. Let's take a screenshot
+for record keeping.
+
 ```{code-cell} ipython3
 nbscreenshot(viewer)
 ```
 
 ## Annotating dividing and non-dividing cells using the points layer
 
-One simple task that a biologist or bioimage analyst might be interseted in annotating each cell as diving or non-dividng.
+One simple task that a biologist or bioimage analyst might be interested in annotating each cell as diving or non-dividing.
```

Comment:
```suggestion
One simple task that a biologist or bioimage analyst might be interested in is annotating each cell as dividing or non-dividing.
```

Reply:
But *diving* cells are so interesting!! 🤣

Reply:
I had a genuine chuckle out of the diving cells, with their swimming caps and everything